perceval/backends/core/meetup.py

```diff
@@ -294,6 +305,9 @@ def setup_cmd_parser(cls):
         group.add_argument('--sleep-time', dest='sleep_time',
                            default=SLEEP_TIME, type=int,
                            help="minimun sleeping time to avoid too many request exception")
+        group.add_argument('--is-oauth-token', dest='is_oauth_token',
+                           action='store_true',
+                           help="True if the token is oauth")
```

Comment:
`True` cannot be passed on the command line so something like `set when using OAuth2 tokens` would work.

Reply:
A new commit updates also the same description in the gitlab backend